go/libraries/doltcore/sqle/enginetest/dolt_queries_diff.go

```diff
@@ -1343,6 +1343,49 @@ inner join t on to_pk = t.pk;`,
 			},
 		},
 	},
+	{
+		Name: "diff on dolt_events schema",
+		SetUpScript: []string{
+			"CREATE TABLE messages (id INT PRIMARY KEY AUTO_INCREMENT, message VARCHAR(255) NOT NULL, created_at DATETIME NOT NULL);",
+			"CREATE EVENT IF NOT EXISTS msg_event ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 1 YEAR DISABLE DO INSERT INTO messages(message,created_at) VALUES('Test Dolt Event 1',NOW());",
+			"CREATE EVENT my_commit ON SCHEDULE EVERY 1 DAY DISABLE DO CALL DOLT_COMMIT('--allow-empty','-am','my daily commit');",
+			"CALL DOLT_ADD('.')",
+			"SET @Commit1 = '';",
+			"CALL DOLT_COMMIT_HASH_OUT(@Commit1, '-am', 'Creating table and events')",
+		},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "SELECT name, definer, execute_every, preserve, status, comment, definition FROM dolt_events;",
```

Comment:
(minor) Looks like this needs the update to pull from `dolt_schemas`? Just double-checking that I understood where everything was stored.